src/convertToObject.js

```diff
@@ -12,7 +12,28 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  let normalized = sourceString;
+
+  while (normalized.includes('  ')
+  || normalized.includes('\n')) {
+    normalized = normalized.replace(/\s\s/g, '');
+    normalized = normalized.replace(/\n/g, '');
+  }
```

Comment:
let's think about time complexity. the `includes` method goes over the entire array (worst case).
the `replace` method does the same. it goes over the entire array. 
just in this while loop you go over the same array 4 times. i don't think it is necessary. 
think about time complexity and how you can do the same thing doing less